src/datachain/lib/udf.py

```diff
@@ -159,6 +159,7 @@ def process(self, file) -> list[float]:
         ```
     """
 
+    is_input_batched = False
```

Comment:
why not use the same `is_output_batch`? (may be rename it to `is_batched)? it seems it always has the same value

Reply:
For generators input will not be batched (single row), but output is batched, thus we need to have separate params for these.